src/main/java/de/tum/in/www1/artemis/domain/PrivacyStatementLanguage.java

```diff
@@ -5,26 +5,24 @@ public enum PrivacyStatementLanguage {
     GERMAN, ENGLISH;
 
     public static PrivacyStatementLanguage fromLanguageShortName(String languageShortName) {
-        if ("de".equals(languageShortName)) {
-            return GERMAN;
-        }
-        else if ("en".equals(languageShortName)) {
-            return ENGLISH;
-        }
-        else {
-            throw new IllegalArgumentException("Language not supported");
-        }
+        return switch (languageShortName) {
```

Comment:
```suggestion
        return switch (languageShortName.toLowerCase()) {
```
Since the check converts to lower case first, it should be done here as well to be consistent.